kibana/Tiltfile

```diff
@@ -0,0 +1,27 @@
+load('ext://helm_remote', 'helm_remote')
+load('ext://namespace', 'namespace_create')
+
+namespace_create('elastic-system')
+
+operator_yaml = local('curl -sSL https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml')
+k8s_yaml(operator_yaml)
+
+k8s_yaml('es.yaml')
+k8s_yaml('kibana.yaml')
+
+k8s_resource(new_name='elasticsearch',
+             objects=['quickstart:elasticsearch'],
+             extra_pod_selectors=[{'elasticsearch.k8s.elastic.co/cluster-name': 'quickstart'}],
+             resource_deps=['elastic-operator'])
+k8s_resource(new_name='kibana',
+             objects=['quickstart:kibana'],
+             extra_pod_selectors=[{'kibana.k8s.elastic.co/name': 'quickstart'}],
+             resource_deps=['elastic-operator', 'elasticsearch'],
+             port_forwards=5601)
+
+local_resource('kibana-password',
```

Comment:
Clever!

Reply:
ya, I feel like a lot of dashboards you set up on kubernetes have a whole auto-generated password dance, and i wonder if we could help people make nicer interfaces around this (for another example, see: https://github.com/tilt-dev/tilt/issues/1851)